client/utils/useDefaultFields.js

```diff
@@ -76,6 +76,24 @@ export const useConnectionsField = (isEditField, fields, connections, onConnecti
   return applyDefaults(type, fields, 'connection', defaults);
 };
 
+export const useMfaField = (isEditField, fields, providers, onProviderChange) => {
+  const type = isEditField ? 'edit' : 'create';
+
+  const defaults = {
+    property: 'multifactor',
+    label: 'MFA Provider',
+    [type]: {
```

Comment:
this syntax looks strange to me. is this some new-fangled ES7 dynamic property syntax?

Reply:
Yes, it's just a computed property name from the ES6 (ES2015) Object Initializer spec: http://www.ecma-international.org/ecma-262/6.0/#sec-object-initializer

Reply:
TIL - I didn't know that was possible.